inventory.schema.json

```
@@ -2788,7 +2788,7 @@
             "firmware": {
               "type": "string"
             },
-            "ip": {
+            "remote_addr": {
               "type": "string",
               "title": "Remote device management IP address"
```

Maybe this should be updated. Not sure of what we should put here.
```suggestion
              "title": "Remote device IP address on which the agent communicates"
```